src/protagonist/API/Converters/AssetConverter.cs

```diff
@@ -419,4 +420,13 @@ public static ImageQuery ToImageQuery(this AssetFilter assetFilter)
 
         return assetFilter;
     }
+
+    /// <summary>
+    /// Converts ImageDeliveryChannels into the old format (WcDeliveryChannels)
+    /// </summary>
+    /// <param name="imageDeliveryChannels"></param>
+    private static string[] ConvertImageDeliveryChannelsToWc(ICollection<ImageDeliveryChannel>? imageDeliveryChannels)
+        => imageDeliveryChannels.IsNullOrEmpty() 
```

Comment:
This method is only called in 1 place, which is is already wrapped in `if (!dbAsset.ImageDeliveryChannels.IsNullOrEmpty())` so it's redundant to have in `ConvertImageDeliveryChannelsToWc`. 

Remove or populate the `/// <param` xml doc element